antora-ui-camel/src/css/toc.css

```diff
@@ -51,11 +51,11 @@
   margin: 0;
 }
 
-.toc .toc-menu li[data-level="2"] a {
+.toc .toc-menu li[data-level='2'] a {
   padding-left: 1.25rem;
 }
 
-.toc .toc-menu li[data-level="3"] a {
+.toc .toc-menu li[data-level='3'] a {
```

Comment:
Style changes in this file should be on a separate commit or in a new pull request.